spec/lib/timberline/config_spec.rb

```diff
@@ -2,6 +2,48 @@
 
 describe Timberline::Config do
   describe "newly created" do
+    context "when the TIMBERLINE_URL env var is defined" do
+      before do
+        ENV["TIMBERLINE_URL"] = "redis://:apassword@ahostname:9000/3?timeout=666&namespace=foobar&sentinel=sentinel1:1&sentinel=sentienl2:2"
+      end
+
+      after do
+        ENV.delete("TIMBERLINE_URL")
+      end
+
+      it "loads the host variable from the env var" do
+        expect(subject.host).to eq("ahostname")
+      end
+
+      it "loads the port variable from the env var" do
+        expect(subject.port).to eq(9000)
+      end
+
+      it "loads the timeout variable from the env var" do
+        expect(subject.timeout).to eq(666)
+      end
+
+      it "loads the password from the env var" do
+        expect(subject.password).to eq("apassword")
+      end
+
+      it "loads the database from the env var" do
+        expect(subject.database).to eq(3)
+      end
+
+      it "loads the namespace from the env var" do
+        expect(subject.namespace).to eq("foobar")
+      end
+
+      it "loads the sentinel servers" do
+        expect(subject.sentinels).to eq([{
+          "host" => "sentinel1", "port" => 1
+        }, {
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.